### PR TITLE
Add Inputs and outputs Angular snippets

### PR DESCRIPTION
**Pull request by PSIB0T**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -234,7 +234,71 @@ Templates in React are writtien inside the JavaScript file using the [JSX langua
 
 # Inputs and Outputs
 
-> TODO
+### Angular
+
+#### Parent Component
```
Comment (feimosi) [inline README.md]: No need for this heading (and the later one)

File: README.md [hunk 2]
```diff
@@ -234,7 +234,71 @@ Templates in React are writtien inside the JavaScript file using the [JSX langua
...
+    <button (click)="emitEditedUser()">
+  `
+})
+
+export class ChildComponent {
+  @Input() user: User;
+  @Output() onEdit: EventEmitter<User>;
+
+  constructor () {
+    this.onEdit = new EventEmitter();
```
Comment (feimosi) [inline README.md]: Could you assign the value where the type definition is?

File: README.md [hunk 3]
```diff
@@ -234,7 +234,71 @@ Templates in React are writtien inside the JavaScript file using the [JSX langua
...
+```js
+@Component({
+  selector: 'child',
+  template: `
+    <p>
+      <input type="text" value="{{user.name}}">
+    </p>
+    <p>
+      <input type="text" value="{{user.email}}">
+    </p>
```
Comment (feimosi) [inline README.md]: Could you remove this `<p>` with email? Just so it's more succinct, without extra clutter.

File: README.md [hunk 4]
```diff
@@ -234,7 +234,71 @@ Templates in React are writtien inside the JavaScript file using the [JSX langua
...
+    <child
+      [user]="user"
+      (onEdit)="editedUser($event)"
+    >
+
+    </child>
+  `
+})
+
+export class ParentComponent {
```
Comment (feimosi) [inline README.md]: We should give these more meaningful names, like `Settings` and the child `UserPreview` (the same goes with selectors).

File: README.md [hunk 5]
```diff
@@ -234,7 +234,71 @@ Templates in React are writtien inside the JavaScript file using the [JSX langua
...
+  user: User;
+  constructor() {
+
+    this.user = {
+      name: 'Foo Bar',
+      email: 'foobar@example.com'
+    }
+
+  }
```
Comment (feimosi) [inline README.md]: Remove extra lines before and after.

**Review by feimosi — CHANGES_REQUESTED** [review]
Thanks for the contribution! Take a look at my comments.

**Comment by PSIB0T** [comment]
I've added all the requested changes :)

**Comment by feimosi** [comment]
Perfect, thanks! :)

**Review by feimosi — APPROVED** [review]

**Pull request merged**